idearium-lib/lib/query.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+/**
+ * Assign some defaults to the passed Mongoose query options.
+ * @param {Object} options Mongoose query options.
+ * @return {Object} Returns the options object for Mongoose queries.
+ */
+const getOptions = (options) => {
```

Comment:
Perhaps you could use [@example](http://usejsdoc.org/tags-example.html) to show how to use the projection functionality in particular?